modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,116 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups provide a way to separate and control access to Anypoint Platform resources such as APIs and applications. Users have access only to the business groups to which they are added.  
```

Comment:
I think "Users have access only to the business groups to which they are added" is slightly misleading because the question that follows is "how do I add a user to a business group", and that makes users think of invites. Can we instead say that users are only members of Business Groups (and therefore show up in the user list) iff they have a role in that BG?